pkgs/applications/networking/instant-messengers/pidgin-plugins/purple-events/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub, pidgin, glib, autoreconfHook, pkgconfig, intltool }:
+
+stdenv.mkDerivation rec {
+  pname = "purple-events";
+  version = "2017-08-25";
+
+  src = fetchFromGitHub {
+    owner = "sardemff7";
+    repo = pname;
+    rev = "7b01b0a2724f675dadf41a8d8e9bee203ef1ccc2";
+    sha256 = "1mhg40swa12inkdhhxf6x9c45h2dijzvdlx1325zgvs2k1vv0a3k";
+  };
+
+  nativeBuildInputs = [ autoreconfHook pkgconfig intltool ];
```

Comment:
In new expressions, `pkg-config` should be used instead of the `pkgconfig` alias.